ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/platform_settings.schema.yml

```
@@ -53,6 +62,13 @@ keys:
             poe:
               type: bool
               default: false
+            per_interface_mtu:
+              type: bool
+              default: true
+              description: |
+                Support for configuration of per interface MTU for p2p links, MLAG SVIs and Network Services.
+                Effectively this means that all settings regarding interface MTU will be ignored if this is false.
+                Platforms without support for per interface MTU can use a single default interface MTU setting. Set this via "interface_default_mtu"
```

The name of the is incorrect here:

```suggestion
                Platforms without support for per interface MTU can use a single default interface MTU setting. Set this via "default_interface_mtu"
```